server/src/weaverbird/backends/mongo_translator/steps/aggregate.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict, List, Union
+
+from weaverbird.pipeline.steps import AggregateStep
+
+
+def translate_aggregate(step: AggregateStep) -> List[Dict[str, Any]]:
+    idblock = {col: f'${col}' for col in step.on}
+    group: Dict[str, Dict] = {}
+    project: Dict[str, Union[str, int]] = {}
+    add_fields = {}
+    group['_id'] = idblock
+
+    for agg_f_step in step.aggregations:
+        cols = agg_f_step.columns
+        new_cols = agg_f_step.new_columns
+
+        if agg_f_step.agg_function == 'count':
+            for i in range(len(cols)):
```

Comment:
What do you thunk of keeping the original comments found in the mongo.ts translator?
I think this one was useful:
```python
// There is no `$count` operator in Mongo, we have to `$sum` 1s to get
// an equivalent result
```